src/tools/wasm-ctor-eval.cpp

```diff
@@ -613,11 +613,23 @@ struct CtorEvalExternalInterface : EvallingModuleRunner::ExternalInterface {
 
   Expression* getSerialization(const Literals& values,
                                Name possibleDefiningGlobal = Name()) {
+    if (values.size() > 1) {
+      // We do not support multivalues in defining globals, which store GC refs.
+      assert(possibleDefiningGlobal.isNull());
+      std::vector<Expression*> children;
+      for (const auto& value : values) {
+        children.push_back(getSerialization(value));
+      }
+      return Builder(*wasm).makeTupleMake(children);
+    }
     assert(values.size() == 1);
     return getSerialization(values[0], possibleDefiningGlobal);
   }
 };
 
+// Whether to emit informative logging to stdout about the eval process.
+static bool quiet = false;
```

Comment:
Another solution might be to add a log prefix and give the lit auto update script an option to specify that prefix so the script can recognize and emit checks for log lines.

Reply:
I see, thanks. That might be useful when we translate the existing tests to lit, which do care about the logging.